sdk/python/kfp/dsl/_container_op.py

```diff
@@ -878,18 +884,36 @@ def add_pod_label(self, name: str, value: str):
         self.pod_labels[name] = value
         return self
 
-    def set_retry(self, num_retries: int, policy: str = None):
+    def set_retry(self,
+                  num_retries: int,
+                  policy: str = None,
```

Comment:
nit: `Optional[str] = None`
the same for other arguments.

Reply:
@chensun @numerology done, thanks!